samples/snippets/pom.xml

```diff
@@ -50,7 +50,7 @@
     <dependency>
       <groupId>com.google.cloud</groupId>
       <artifactId>google-iam-admin</artifactId>
-      <version>3.49.0</version>
+      <version>3.50.0</version>
```

Comment:
Noticed this PR accidentally. Do we need to specify version for iam-admin? I see that it is already [part of google-cloud-java](https://github.com/googleapis/google-cloud-java/tree/main/java-iam-admin).